fv3core/tests/main/test_memoize_storages.py

```diff
@@ -26,36 +23,10 @@ def init(request):
         raise NotImplementedError(request.param)
 
 
-# having backend as an arg for tests means the test runs on each backend
-
-
-def test_storage_is_cached(backend, shape, origin, init):
-    outputs = []
-    for _ in range(2):
-        outputs.append(
-            make_storage_from_shape(
-                shape,
-                origin=origin,
-                init=init,
-                cache_key="test-cached",
-                backend=backend,
-            )
-        )
-    assert outputs[0] is outputs[1]
-
-
 def test_uncached_storage_is_not_cached(backend, shape, origin, init):
     outputs = []
     for _ in range(2):
         outputs.append(
-            make_storage_from_shape_uncached(
-                shape, origin=origin, init=init, backend=backend
-            )
+            make_storage_from_shape(shape, origin=origin, init=init, backend=backend)
```

Comment:
You can delete this test file - its purpose was to test the caching system, and now that you've deleted it there's no worry that this test would fail.